### PR TITLE
Addition of a new FCU

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
AC unit with heating and cooling (zone temp control) @kevin-hereworks

### CODE REVIEW

**Comment by Ciaran-Oflannagain** [comment]
@cstirdivant - Please see FCU PR (AC unit with heating and cooling (zone temp control)), when you have a moment. Thank you. Ciaran

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 1]
```diff
@@ -19,6 +19,17 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_SS_CO2M_ZTC_STC_RTM:
+  description: "AC unit with heating and cooling (zone temp control)"
+  is_canonical: true
+  implements:
+  - FCU
+  - SS
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:27]: This is ambiguous - is this SS control for the fan? The cooling/heating element?

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml [hunk 2]
```diff
@@ -19,6 +19,17 @@
...
+  - CO2M
+  - ZTC
+  - STC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/FCU.yaml:30]: How can the unit control to both supply and zone temp?

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 2 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml]: Apologies, this was also just supply temp monitoring not control.

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:27]: Yes the SS is for the entire unit

Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:27]: I would expect there to be separate SS for the fan (DFSS, etc) and the conditioning elements (DXZTC, HTZTC, etc)

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/FCU.yaml (hunk 1 above)
Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/FCU.yaml:27]: Okay thanks a million - we only have a single run command and status from the bms points but I will check them. Thank you

**Review by cstirdivant — CHANGES_REQUESTED** [review]

**Pull request closed without merging**